index.html

```diff
@@ -0,0 +1,176 @@
+              <img src="./images/motivation.png" alt="motivation-icone" />
+              <span>Motivation</span>
+            </div>
+            <div class="right">
+              Listen to the motivation form various personalities about the
+              message of sharing and opening opportunities to the less
+              privileged in your community.
+            </div>
+          </div>
+          <div class="program-card">
+            <div class="left">
+              <img src="./images/forum.png" alt="forum-icone" />
+              <span>Forum</span>
+            </div>
+            <div class="right">
+              Have the time to share your toughts and opinions with our team for
+              each topic.
+            </div>
+          </div>
+          <div class="program-card">
+            <div class="left">
+              <img src="./images/lecture.png" alt="lecture-icon" />
+              <span>Donation</span>
+            </div>
+            <div class="right">
+              The value of life is not in its duration, but in its donation.
+              What a privilege to be here on the planet to contribute your
+              unique donation to humankind!
+            </div>
+          </div>
+          <div class="program-card">
+            <div class="left">
+              <img src="./images/support.png" alt="support-icone" />
+              <span>Support</span>
+            </div>
+            <div class="right">
+              Whatever little it maybe, it means a lot to someone out there.
+              Support the needy in our community.
+            </div>
+          </div>
+          <div class="program-card">
+            <div class="left">
+              <img src="./images/workShop.png" alt="workshop-icone" />
+              <span>Workshop</span>
+            </div>
+            <div class="right">
+              Get oppotunities to network with GC affiliates around the globe.
+            </div>
+          </div>
+        </div>
+        <div class="join-btn">
+          <a href="#">SEE THE WHOLE PROGRAM</a>
+        </div>
+      </section>
+      <section id="sponsor" class="sponsor"></section>
+      <section class="partner">
+        <h3 class="partner-title">Partner</h3>
+        <div class="partner-container">
+          <figure>
+            <img src="./images/mozilla.png" alt="img1" />
+          </figure>
+          <figure>
+            <img src="./images/google.png" alt="img2" />
+          </figure>
+          <figure>
+            <img src="./images/naver.png" alt="img3" />
+          </figure>
+          <figure>
+            <img src="./images/airbnb.jpg" alt="img4" />
+          </figure>
+        </div>
+      </section>
+      <div class="last">
+        <img src="./images/creative-commons-license-logo.jpeg" alt="logo" />
+        <div>
+          <p>&copy; 2022 E-learning. Some Rights Reserved.</p>
+          <p>Created by Ilham Bouaik</p>
+        </div>
+      </div>
```

Comment:
- [ ] Kindly try to match the Behance design in the `footer` section on the home page mobile version by increasing the top and bottom spacing and increasing the gap between the logo and text.

| Your design | Behance design |
| --- | --- | 
| ![image](https://user-images.githubusercontent.com/24703171/164565767-f5ce2747-5099-4536-b335-442d9ee7c7e7.png) | ![image](https://user-images.githubusercontent.com/24703171/164565706-3cbfcd63-aade-4fff-9001-8852575897a7.png) |

- [ ] and in the desktop version decrease the logo size a little bit.
- [ ] Kindly change the `div` tag to be `footer` tag for semantic purposes.